src/plugins/imagery/components/ImageryViewLayout.vue

```diff
@@ -66,7 +66,7 @@ export default {
     data() {
         return {
             autoScroll: true,
-            date: '',
+            bounds: {},
```

Comment:
I don't *think* there's any reason to track bounds locally? We can use the canonical bounds provided by `openmct.time.bounds()`, which are guaranteed to always be up to date. The downsides of tracking it locally are:
1. It could get out of sync with `openmct.telemetry.bounds()`
2. Defining it as data on a Vue component has some overhead. Vue will make it reactive and run some code every time it's updated (could be hundreds of times a second in realtime mode).

Reply:
Makes sense, removed bounds locally, anytime I need bounds I grab from openmct.telemetry.bounds()